src/SwipeableListItem.js

```diff
@@ -58,6 +62,8 @@ class SwipeableListItem extends PureComponent {
   }
 
   componentWillUnmount() {
+    cancelAnimationFrame(this.requestedAnimationFrame);
```

Comment:
Please guard to always use a valid and non-null `requestId`. Consider using this pattern to clear already scheduled frame callback:

```suggestion
    if (this.requestedAnimationFrame) {
      cancelAnimationFrame(this.requestedAnimationFrame);
      
      this.requestedAnimationFrame = null;
    }
```